src/lib/dcob.rb

```diff
@@ -86,6 +104,11 @@ def verify_signature(payload_body)
     puts "You must specify cfg.dco_info_url in config.toml"
     exit 1
   end
-end
 
-Dcob::Server.run!
+  if config["cfg"]["dco_webhook_url"]
+    DCO_WEBHOOK_URL = config["cfg"]["dco_webhook_url"]
+  else
+    puts "You must specify cfg.dco_webhook_url in config.toml"
```

Comment:
I wonder if this really needs to be configurable. The webhook URL (e.g. "me" from the bot's perspective) could be determined from the incoming `request.url` or `request.scheme + '://' + request.host + '/payload'`